README.md

```diff
@@ -51,6 +54,13 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
+
+var pizza {
+  temperature: 70,
+  bake: function () {
+    temperature = 300;
```

Comment:
this line creates a global variable called `temperature`. If you want to modify the property, you need to reference it on the object using this:
`this.temperature = 300;`